### ISSUE TITLE
Interactive Window - No sys info at the top when kernel doesn't launch

### ISSUE THREAD

**Issue by rchiodo** [microsoft/vscode#127530]
Testing #127378

If my kernel doesn't start (because I either picked a kernel without ipykernel installed or it's not working) I don't get sysinfo before cells:

![image](https://user-images.githubusercontent.com/19672699/123850032-596b5100-d8ce-11eb-86fd-869988286ab0.png)

**Comment by joyceerhl** [comment]
@rchiodo If there's no kernel connection there's no sys info to display. Would you expect to see a message indicating that the kernel is dead?

**Comment by rchiodo** [comment]
The old window didn't let you add cells without a sysinfo. Cause it would be weird if suddenly the sysinfo appeared after those two cells. What kernel did they run with?

**Comment by joyceerhl** [comment]
Yeah this comes back to the interactive window `NotebookDocument` being manipulatable without a controller necessarily being 'ready', where the static controller registration is divorced from the underlying kernel being ready

**Comment by joyceerhl** [comment]
The existing webview interactive window has the exact same problem. This is not peculiar to the native interactive window 

![image](https://user-images.githubusercontent.com/30305945/124018803-1a580100-d99d-11eb-83aa-7c92911a12d9.png)

![image](https://user-images.githubusercontent.com/30305945/124018783-1330f300-d99d-11eb-9ebc-bcba41a420ff.png)
